src/vSphereServerCollector.class.inc.php

```diff
@@ -90,6 +90,8 @@ protected static function DoCollectServer($aHyperV)
 			'osversion_id' => $aHyperV['osversion_id'],
 			'cpu' => $aHyperV['cpu'],
 			'ram' => $aHyperV['ram'],
+			'serialnumber' => $aHyperV['serialnumber'],
+			'managementip' => $aHyperV['managementip'],
```

Comment:
Should maybe have a check if TeemIP is installed to differentiate between `managmentip_id` and `managementip`?